### PR TITLE
294 - breadcrumb nav on explore pages

### PR DESCRIPTION
**Pull request by nick-lennox**
**Issue**

issue #294 
"When I am on a course page or subject page I want to know relatively where I am on the YACS page."

*Example:*
Added breadcrumb navigation on all subpages of /explore

**Test Procedure**

Go to YACS home -> explore, notice the row below the header that tells you where you are relative to the rest of the YACS pages

**Photos**

Screen Shot 2020-11-15 at 9 02 01 PM

### CODE REVIEW

**Review by 123joshuawu — CHANGES_REQUESTED** [review]
Sweet, looks really nice.

Just two things:
1) I think we don't need `Home` in the breadcrumb because `Explore` is the root page for all the explore pages
2) Can we use `to` instead of `href` for the links? This will avoid reloading the page when using the breadcrumbs

**Comment by nick-lennox** [comment]
> 1. I think we don't need `Home` in the breadcrumb because `Explore` is the root page for all the explore pages

I feel like it may not be a bad idea to leave it in to make the navigation back to the main page a bit easier (we also don't necessarily need to call it 'Home'), but if you feel strongly about it I can definitely change it. For sure I'll change the hrefs though.

Thanks for the feedback!

**Review by 123joshuawu — APPROVED** [review]
Sounds good to me

**Comment by marchdan** [comment]
@nick-lennox @123joshuawu What about if we used "YACS" instead of "Home"? I do agree that Explore is kinda a root page, but explore is still a branch off of the root homepage. this is just a thought, I think either way works fine

**Comment by nick-lennox** [comment]
> @nick-lennox @123joshuawu What about if we used "YACS" instead of "Home"? I do agree that Explore is kinda a root page, but explore is still a branch off of the root homepage. this is just a thought, I think either way works fine

I was thinking this would be a good solution too.

**Comment by marchdan** [comment]
@nick-lennox Please resolve merge conflicts so this can be reviewed

File: src/web/src/pages/SubjectExplorer.vue [hunk 1]
```diff
@@ -1,78 +1,78 @@
...
-            {{ course.title }}
-            <br />
-            {{ course.department }}
-            {{ course.level }}
-            <br />
-          </b-button>
-        </b-col>
-      </b-row>
-    </b-col>
+      <!-- left column of courses -->
```
Comment (marchdan) [inline src/web/src/pages/SubjectExplorer.vue:12]: Is there a specific reason for the extra spacing here?

**Comment by marchdan** [comment]
Is there a specific reason for the extra spacing change in this file?

https://github.com/YACS-RCOS/yacs.n/pull/328/files#diff-a09f667c120da1ee7f101c7058309ff0ed6bbfd9432a1077ad7c0cd480d00257

**Review by marchdan — APPROVED** [review]
Looks good. Course explorer seems to have some weird loading issues, but I think there is a separate PR to fix that

**Comment by marchdan** [comment]
@123joshuawu Could you double-check this? Looks good to me, but there have been changes since you approved

**Review by 123joshuawu — APPROVED** [review]

**Pull request merged**